lib/frame.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class Frame
+  attr_reader :rolls
+
+  def initialize
+    @rolls = []
+  end
+
+  def roll(user_input)
+    @rolls << user_input
+  end
+
+  def is_strike?
+    @rolls.first == 10
+  end
+
+  def is_spare?
+    @rolls.length == 2 && count == 10
+  end
+
+  def count
```

Comment:
Is this method necessary, or would it be just as good to call `@rolls.sum` directly above? Unless it also needs to be called by another class below, in which case, that makes sense!

Reply:
I was thinking about this as well, I guess this one isn't necessary, it might be more DRY, as I use it to calculate bonuses, where my function now does not need to do the rolls.sum